plugins/deploy_env_vars/app/views/samson_deploy_env_vars/_deploy_form.html.erb

```diff
@@ -0,0 +1,33 @@
+<fieldset>
+  <legend>
+    Specific Environment Variables for this deploy <%= additional_info SamsonEnv::HELP_TEXT %>
+  </legend>
+
+  <% current_vars = form.object.environment_variables %>
+  <%= form.fields_for :environment_variables, current_vars << EnvironmentVariable.new do |fields| %>
```

Comment:
prefer `(current_vars << EnvironmentVariable.new)` to make it less like the `<<` is using the `do |fields|`
... maybe better to just do it in the line above ... and not do it if the last one is `.new_record?` already